index.html

```diff
@@ -20,7 +20,7 @@
       <div class="content-en">
         <h2>Cat (en)</h2>
         <p>
-          The <span class="wmf-wp-with-preview" data-wikipedia-preview>cat</span> (Felis catus) is a small <span class="wmf-wp-with-preview" data-wikipedia-preview>carnivorous</span> mammal. It is the only domesticated species in the family Felidae and often referred to as the domestic cat to distinguish it from wild members of the <span class="wmf-wp-with-preview" data-wikipedia-preview>family</span>. The cat is either a house cat or a farm cat, which are pets, or a feral cat, which ranges freely and avoids human contact. A house cat is valued by humans for companionship and <span class="wmf-wp-with-preview" data-wikipedia-preview>for (not an article)</span> its ability to hunt rodents. About 60 cat breeds are recognized by various cat registries.
+          The <span class="wmf-wp-with-preview" data-wikipedia-preview>cat</span> (Felis catus) is a small <span class="wmf-wp-with-preview" data-wikipedia-preview>carnivorous</span> mammal. It is the only domesticated species in the family <a href="https://en.wikipedia.org/wiki/Felidae">Felidae</a> and often referred to as the domestic cat to distinguish it from wild members of the <span class="wmf-wp-with-preview" data-wikipedia-preview>family</span>. The cat is either a house cat or a farm cat, which are pets, or a feral cat, which ranges freely and avoids human contact. A house cat is valued by <a href="https://en.wikipedia.org/wiki/humans">humans</a> for companionship and <span class="wmf-wp-with-preview" data-wikipedia-preview>for (not an article)</span> its ability to hunt rodents. About 60 cat breeds are recognized by various cat registries.
```

Comment:
this is the temporarily a tag solution for this demo purpose, ultimately this demo page will be replaced with another demo page.